packages/core/src/components/Button/ButtonConstants.ts

```diff
@@ -26,19 +26,6 @@ export const OLD_BUTTON_SIZES = {
 
 export type Size = (typeof SIZES)[keyof typeof SIZES] | keyof typeof OLD_BUTTON_SIZES;
 
-// Support old sizes (backward compatible)
-export const getActualSize = (size?: Size) => {
```

Comment:
No need to add it to the change log?

Reply:
I don't think anyone knew about it but will add anyway 😅